en/DotNETPluginDevelopment/APIs/Namespace/LLNET.AllowList/Class/AllowListManager/AllowListManager.md

```diff
@@ -9,7 +9,7 @@
 <br>
 
 C#
-```cs
+```csharp
```

Comment:
Please do not modify the language code, which conforms to the specifications of Prism.js and keeps a simple form.